Actors/CounterActor/CounterActorTestClient/testclient.sh

```diff
@@ -10,5 +10,17 @@ check_errs()
 
 DIR=`dirname $0`
 export LD_LIBRARY_PATH=/opt/microsoft/servicefabric/bin/Fabric/Fabric.Code/
-dotnet $DIR/CounterActorTestClient.dll $@
+
+. /etc/os-release
+linuxDistrib=$ID
+if [ $linuxDistrib = "rhel" ]; then
+  source scl_source enable rh-dotnet20
+  exitCode=$?
+  if [ $exitCode != 0 ]; then
+        echo "Failed: source scl_source enable rh-dotnet20 : ExitCode: $exitCode"
+        exit $exitCode
+  fi
+fi
+
+exec dotnet $DIR/CounterActorTestClient.dll "$@"
```

Comment:
Do we need exec? The yo PR doesn't have the exec command.

Reply:
Will check that and see